config/routes.rb

```diff
@@ -8,6 +8,10 @@
 
   root 'home#index'
 
+  #temporary redirections
+  get '/founders', to: 'projects#index', action: :index
+  get '/wealth', to: 'projects#index', action: :index
+
```

Comment:
Temporary route definitions for showcase purposes. These will be removed once we have the founders and wealth entities/pages built: https://trello.com/c/EytINkcY/63-remove-temporary-redirections-for-founders-and-wealth